.circleci/config.yml

```diff
@@ -97,7 +98,26 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - run: ./do package
+      - run:
+          name: "Install signing keys"
+          command: |
+            exec 2>/dev/null
+
+            echo "Importing signing keys"
+            echo -n "${SIGNING_KEY_ENCODED}" | base64 --decode >signing_key_decoded.key
+            gpg --batch --yes --passphrase "${SIGNING_KEY_PASSPHRASE}" --import signing_key_decoded.key
+            rm signing_key_decoded.key
+            curl https://keys.openpgp.org/vks/v1/by-fingerprint/"${GPG_ID}" >pub-key.asc
```

Comment:
Does this work as expected? I thought we'd need something like

```
curl -q URL -o pub-key.asc
```

Reply:
Yes: https://github.com/circleci/circleci-runner/blob/1e28eb258b3ba2020f7ca992049a6ec86d68d07f/.circleci/config.yml#L971

https://app.circleci.com/pipelines/github/CircleCI-Public/container-runner-helm-chart/407/workflows/3dd7d6a6-10bf-44eb-b27b-0b979d0efb8e/jobs/1192?invite=true#step-104-285_92